classes/class-twenty-twenty-one-customize.php

```diff
@@ -146,6 +146,30 @@ public function register( $wp_customize ) {
 					)
 				)
 			);
+
+			$wp_customize->add_setting(
+				'respect_user_color_preference',
+				array(
+					'capability'        => 'edit_theme_options',
+					'default'           => true,
+					'sanitize_callback' => function( $value ) {
+						return (bool) $value;
+					},
+				)
+			);
+
+			$wp_customize->add_control(
+				'respect_user_color_preference',
+				array(
+					'type'            => 'checkbox',
+					'section'         => 'colors',
+					'label'           => esc_html__( 'Respect visitor\'s device light/dark mode settings', 'twentytwentyone' ),
+					'description'     => __( 'Show your site in dark mode if a visitor to your site requests it.', 'twentytwentyone' ),
```

Comment:
I think this could be more descriptive — it says basically the same thing as the label, and doesn't give the site owner any info on how to enable it themselves. Thinking on this more, but I don't want to hold up my review.

Reply:
```suggestion
					'label'           => esc_html__( 'Respect visitor\'s device dark mode settings', 'twentytwentyone' ),
					'description'     => __( 'Dark mode is a device setting. If a visitor to your site requests it, your site will be shown with a dark background and light text.', 'twentytwentyone' ),
```

label: Removed the "light/" — technically we don't respect a "light mode", if the user prefers light we don't change dark backgrounds.
description: Made it a little more clear what happens, and re-emphasized the device-setting nature of dark mode.

Reply:
Makes sense :+1: